Tests/WalletTests.swift

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2017 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import TrezorCrypto
+@testable import TrustKeystore
+import XCTest
+
+class WalletTests: XCTestCase {
+    let words = "ripple scissors kick mammal hire column oak again sun offer wealth tomorrow wagon turn fatal"
```

Comment:
Can you add test with optional password? is it empty string by default?

Reply:
Done